apps/api/src/migration/1741307582814-AddUnitColumnToAllowedActivity.ts

```diff
@@ -0,0 +1,118 @@
+import { MigrationInterface, QueryRunner, TableColumn, TableUnique } from 'typeorm';
+
+export class AddUnitColumnToAllowedActivity1741307582814 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const table = await queryRunner.getTable('allowed_activity');
```

Comment:
If we are considering cases where there may not be a allowed_activity table or there is a missing column, we should add some logging to this migration so we can have a trace of it.

Reply:
changed the conditional to assert.